projects/distributed-tracing/src/shared/components/span-detail/span-detail.component.ts

```diff
@@ -23,30 +24,30 @@ import { SpanDetailLayoutStyle } from './span-detail-layout-style';
         <ng-content></ng-content>
       </div>
 
-      <ht-tab-group class="tabs-group">
-        <ht-tab label="Request" *ngIf="this.showRequestTab">
+      <ht-tab-group class="tabs-group" [activeLabel]="this.activeLabel">
+        <ht-tab label="${SpanDetailTab.Request}" *ngIf="this.showRequestTab">
           <ht-span-request-detail
             class="request"
             [layout]="this.layout"
             [requestHeaders]="this.spanData.requestHeaders"
             [requestBody]="this.spanData.requestBody"
           ></ht-span-request-detail>
         </ht-tab>
-        <ht-tab label="Response" *ngIf="this.showResponseTab">
+        <ht-tab label="${SpanDetailTab.Response}" *ngIf="this.showResponseTab">
           <ht-span-response-detail
             class="response"
             [layout]="this.layout"
             [responseHeaders]="this.spanData.responseHeaders"
             [responseBody]="this.spanData.responseBody"
           ></ht-span-response-detail>
         </ht-tab>
-        <ht-tab label="Attributes" class="attributes">
+        <ht-tab label="${SpanDetailTab.Attributes}" class="attributes">
           <ht-span-tags-detail [tags]="this.spanData.tags"></ht-span-tags-detail>
         </ht-tab>
-        <ht-tab label="Exit Calls" *ngIf="this.showExitCallsTab">
+        <ht-tab label="${SpanDetailTab.ExitCalls}" *ngIf="this.showExitCallsTab">
           <ht-span-exit-calls [exitCalls]="this.spanData.exitCallsBreakup"></ht-span-exit-calls>
         </ht-tab>
-        <ht-tab *ngIf="this.showLogEventstab" label="Logs" [badge]="this.totalLogEvents">
+        <ht-tab *ngIf="this.showLogEventstab" label="${SpanDetailTab.Logs}" [badge]="this.totalLogEvents">
```

Comment:
super nit : `T` should be capital for tab in the variable name `showLogEventsTab`

Reply:
Done